src/lib.rs

```diff
@@ -52,18 +84,61 @@ impl<T: 'static + Send + Sync> Default for DefaultRaycastingPlugin<T> {
 pub enum RaycastSystem {
     BuildRays,
     UpdateRaycast,
-    UpdateDebugCursor,
+    #[cfg(feature = "debug")]
+    AddOrRemoveDebugCursors,
+    #[cfg(feature = "debug")]
+    EnableOrDisableDebugCursors,
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum Enabled {
+    /// Ray casting enabled
+    Enabled,
+    /// Ray casting disabled
+    Disabled,
+}
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum Debug {
```

Comment:
To mirror the above comment, it might make sense to name this `DebugState` to make them symmetric.

Reply:
Done